README.md

```diff
@@ -11,4 +11,5 @@ Chris Dao showcasing how to use GitHub's branch function using dummy file.
 
 Chris Dao showcasing how to use GitHub's branch function using dummy file. 
 
-#Hello World
```

Comment:
This line specifically showcases, in merged view, how the differences are highlighted - split view and merged views are completely preferences, depending on end-user.

Reply:
Here, we demonstrate how a team maintains version control in the branch and effectively communicate thru leaving "review comments" - the emojis are a fun way to create polling systems or a fun way to provide/request feedback.